File: VleisurePartner.Web/Models/RequestModels/HotelListRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using VleisurePartner.Web.Infrastructure.Typescript;

namespace VleisurePartner.Web.Models.RequestModels
{
    public class HotelListRequest : GenericRequest, ITypeProxy
    {
        public string ArrivalDate { get; set; }
        public string DepartureDate { get; set; }
        public string LanguageCode { get; set; }
        public string CityCode { get; set; }
        public IEnumerable<int> HotelIds { get; set; }
        public IEnumerable<RoomGuestRequestModel> RoomGuests { get; set; }

     
    }
}

File: VleisurePartner.Web/src/proxy/HotelListRequest.ts

/* tslint:disable */
declare namespace ProxyModel{
    
    export interface HotelListRequest extends GenericRequest {
        
        arrivalDate: string;
        departureDate: string;
        languageCode: string;
        cityCode: string;
        hotelIds: number[];
        roomGuests: RoomGuestRequestModel[];
        }
    
}